src/chat/channel.py

```diff
@@ -308,10 +308,8 @@ def printRaw(self, name, text, scroll_forced=False):
         '''
         Print an raw message in the chatArea of the channel
         '''
-        if self.lobby.client.players.isPlayer(name):
-            id = self.lobby.client.players[name].id
-        else:
-            id = -1
+        id = self.lobby.client.players.getID(name)
```

Comment:
I'm thinking we may want to make the players thing global at some point: these parent pointers are gnarly as hell.

Perhaps we'll even want to go all model/view on this and make the players system a Qt Model at some point.